compose.yml

```diff
@@ -0,0 +1,21 @@
+# Example of a Docker Compose file that runs the DNS and Nginx containers together
+
+services:
+  dns:
+    build: ./dns
+    restart: always
+    ports:
+      - 53:53/udp
+    env_file:
+      - ./dns/.env
+
+  nginx:
+    build: ./nginx
+    restart: always
+    ports:
+      - 80:80
```

Comment:
This is specifically made for SSL, does port 80 need to be used?

Reply:
I think so, since conntest never uses SSL.

Reply:
This is fair, I forgot about this 👍